scripts/download-win-lib.js

```diff
@@ -0,0 +1,13 @@
+var download = require('./download').download;
+var path = require('path');
+var fs = require('fs');
+
+var TAR_URL = 'https://github.com/nteract/zmq-prebuilt/releases/download/win-libzmq-4.1.5-v140/libzmq-' + process.arch + '.lib';
```

Comment:
`apm` uses a different logic to determine the architecture: https://github.com/atom/apm/blob/master/src/apm.coffee#L78

Is `process.arch` OK in our case?